src/lib/intent/index.ts

```diff
@@ -20,5 +21,6 @@ export {
   FieldCreateIntent as FieldCreate,
   ContentTypeCreateIntent as ContentTypeCreate,
   ContentTypeDeleteIntent as ContentTypeDelete,
-  ContentTypeUpdateIntent as ContentTypeUpdate
+  ContentTypeUpdateIntent as ContentTypeUpdate,
+  EntryDeriveIntent as EntryDerive
```

Comment:
Don't we have to also export the content transform intent? 🤔

Reply:
Isn't this `EntryTransformIntent as EntryTransform`?